### PR TITLE
Contraction f16, bf16, f32_f16, f32_bf16, f64_f32

### PR DESCRIPTION
**Pull request by CongMa13**

### CODE REVIEW

File: test/01_contraction/configs/bilinear_test_params.yaml
Comment (CongMa13) [inline test/01_contraction/configs/bilinear_test_params.yaml:14]: actor critic does not work since the specified instances cannot process the input data after ck team fixed the `IsSupportedArgument()`

File: test/utils.hpp
Comment (CongMa13) [inline test/utils.hpp]: epsilone of type is not a good option to measure the difference between data.

epsilone<double> = 2.22045e-14 , which is too small

Comment (CongMa13) [reply test/utils.hpp]: Data Type Epsilon

f16: 0.000976562
bf16: 0.0078125
float: 1.19209e-07
double: 2.22045e-16

bf16::epsilon() > 0.1% and we cannot reduce it. So, use 0.1% and the default tolerance.

File: library/include/hiptensor/internal/xfloat32.hpp
Comment (cgmillette) [inline library/include/hiptensor/internal/xfloat32.hpp:1]: CK doesn't support xfloat32, so let's remove this type

File: README.md
Comment (cgmillette) [inline README.md]: gfx90a

**Review by cgmillette — DISMISSED** [review]
Looks good to me - let's wait until CK merges and then we will have to update CI to the latest

**Review by cgmillette — DISMISSED** [review]

**Review by cgmillette — APPROVED** [review]
LGTM

**Pull request merged**
